src/helpers/formatBookingId.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-const formatBookingId = (bookingDatabaseId: number): string => {
+const formatBookingId = (bookingDatabaseId: ?number): string => {
```

Comment:
> String(undefined).split('').reverse().join('').match(/.{1,3}/g).join(' ').split('').reverse().join('')

`'und efi ned'`

I guess it was required on purpose 😎

Reply:
btw does this have unit test? because it should have unit test...

Reply:
https://github.com/kiwicom/smart-faq/issues/336